### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by tshin7**
I fixed all of the warnings except for the "No duplicate props allowed" in src/components/Map/Map.js
The warning is about the two ref props in the GoogleMap component. I wasn't sure what props.onMapLoad and props.mapRef were.

### CODE REVIEW

**Comment by galiat** [comment]
NIce! @tshin7 - rather than comment out things, could you delete them? Looks like its code we dont need.

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -5,12 +5,7 @@ import {
   NavbarToggler,
   NavbarBrand,
   Nav,
-  NavItem,
-  NavLink,
```
Comment (Khoawala) [inline src/components/Header/Header.js]: Keep NavLink

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -37,7 +32,7 @@ class Header extends Component {
       <div>
         <Navbar color="light" light expand="md">
         <NavbarBrand className="Logo-icon">
-        <a className="Logo-icon" href="https://communityconnect.netlify.com/"><img src={CClargelogo}/></a>
+        <a className="Logo-icon" href="https://communityconnect.netlify.com/"><img src={CClargelogo} alt="Community Connect logo"/></a>
```
Comment (Khoawala) [inline src/components/Header/Header.js]: The html <a> tag is outputting a major warning in the console. "validateDOMNesting(...): <a> cannot appear as a descendant of <a>."  Instead just put href="/" as a NavbarBrand property to create the link to the main page. Documentation: https://reactstrap.github.io/components/navbar/

**Comment by tshin7** [comment]
@AsstroGrill

**Comment by tshin7** [comment]
> NIce! @tshin7 - rather than comment out things, could you delete them? Looks like its code we dont need.

@galiat @Khoawala Everything should be good now.

**Pull request merged**